container/nginx/jwt_pub.key

```diff
@@ -0,0 +1,4 @@
+-----BEGIN PUBLIC KEY-----
```

Comment:
This is the staging public key.

Reply:
that's reasonable. ultimately, it's a public key so security shouldn't need to be that tight.